requirements/requirements-gguf_editor_gui.txt

```
@@ -0,0 +1,8 @@
+numpy~=1.26.4
+PySide6~=6.9.0
+PySide6_Addons~=6.9.0
+PySide6_Essentials~=6.9.0
+PyYAML~=6.0.1
+sentencepiece~=0.2.0
+shiboken6~=6.9.0
```

```suggestion
PySide6~=6.9.0
PyYAML~=6.0.1
sentencepiece~=0.2.0
```
These are dependencies of the main package since PySide6 6.3.0